src/app-layout/__integ__/runtime-drawers.test.ts

```diff
@@ -77,9 +77,14 @@ describe.each(['classic', 'refresh', 'refresh-toolbar'] as const)('%s', theme =>
 });
 
 describe('Visual refresh toolbar only', () => {
-  function setupTest(testFn: (page: BasePageObject) => Promise<void>) {
+  class PageObject extends BasePageObject {
+    getBrowser() {
+      return this.browser;
```

Comment:
the `browser` property is protected in the `BasePageObject` class and it needs to be public here. `get browser` does not work as typescript considers it as overriding and throws `TS2611`

Reply:
addressed